.github/workflows/dotnet.yml

```diff
@@ -126,42 +126,42 @@ jobs:
         with:
           dotnet-version: 6.0.x
 
-  approve_staging_or_uat_release: 
-    runs-on: ubuntu-latest 
-
-    steps:
-      - uses: trstringer/manual-approval@v1
-        with:
-          secret: ${{ github.TOKEN }}
-          approvers: Thumbeza
```

Comment:
We don't have approval rights, Might be because your are still the only approve listed.

Reply:
The approve button is in the review dropdown under changed files